tests/conftest.py

```diff
@@ -146,3 +148,23 @@ def pytest_runtest_setup(item):
     skip_altcoins = int(os.environ.get("TREZOR_PYTEST_SKIP_ALTCOINS", 0))
     if item.get_closest_marker("altcoin") and skip_altcoins:
         pytest.skip("Skipping altcoin test")
+
+
+@pytest.hookimpl(tryfirst=True, hookwrapper=True)
+def pytest_runtest_makereport(item, call):
+    # Make test results available in fixtures.
+    # See https://docs.pytest.org/en/latest/example/simple.html#making-test-result-information-available-in-fixtures
+    outcome = yield
+    rep = outcome.get_result()
+    setattr(item, f"rep_{rep.when}", rep)
```

Comment:
I'm unsure what is this doing and how it relates to the the click tests.

Reply:
it makes the result of the test available in a fixture ;)

The `device_handler` fixture raises an error if you forget to check status of your background task. However, if your test fails, it probably never gets to the line that does the checking. In that case you'd get a failure *and* an error. That seemed less than friendly, so instead the error is only raised if the test succeeds.

Reply:
I didn't realize what is a fixture in this context. Thanks.

Reply:
added a comment that says a little about this :)